languages/rust/oso/src/builtins.rs

```diff
@@ -29,7 +29,114 @@ fn dictionary() -> Class<HashMap<Symbol, Value>> {
 fn string() -> Class<String> {
     Class::<String>::with_default()
         .name("String")
+        .add_method("len", |s: &String| s.len() as i64)
```

Comment:
Needed because we don't implement `From/ToPolar` for `usize`? We should add those, then this can be `String::len`

Reply:
Since we use an `i64` for ints, a `u64` (or `usize`) doesn't actually fit. Seems better to just have these methods return an int instead of *say* we support `usize` when we don't exactly.

Reply:
Yeah, separate issue for us to fix with polar-core.